src/main/java/core/levels/LevelManager.java

```diff
@@ -37,12 +44,13 @@ public LevelManager(LevelState level) {
 
         this.batch = new SpriteBatch();
 
+        this.levelDifficultyToEnemies = createLevelDifficultyToEnemies();
+        // create enemy list based on difficulty of level
+        this.enemySpawns = createEnemyList(levelDifficultyToEnemies.get(level.getLevelDifficulty()));
```

Comment:
this needs to be more abstract – should be able to spawn more than just enemies, and name should reflect this